server.js

```diff
@@ -15,3 +15,5 @@ require('./models/init')().then(function () {
 }).catch(function(err) {
   console.log(err);
 });
+
+require('./adapters/software');
```

Comment:
This is a test require from my branch

Why the f* almost everybody never check the files they `git add` before they `git commit` ?

`git status` and `git diff` are our friends